lib/gtfs/route_pattern.ex

```diff
@@ -37,4 +37,7 @@ defmodule Gtfs.RoutePattern do
       representative_trip_id: row["representative_trip_id"]
     }
   end
+
+  @spec member_of_routes?(MapSet.t(id()), %{required(String.t()) => String.t()}) :: boolean
+  def member_of_routes?(route_ids, row), do: Enum.member?(route_ids, row["route_id"])
```

Comment:
- Since this is a function on the route_pattern, the route_pattern (here, `row`) should be the first argument: maybe naming it something like `in_id_set?(row, set)` makes that clearer?
- You can ensure the argument is a MapSet (per the type) with `(row, %MapSet{} = route_ids)` and then simplify the body to `MapSet.member?(route_ids, row["route_id"])`